charts/kong/example-values/minimal-k4k8s-with-kong-enterprise.yaml

```diff
@@ -0,0 +1,54 @@
+# Basic values.yaml for Kong for Kubernetes with Kong Enterprise
+# Several settings (search for the string "you-must") require user-provided
+# Secrets. These Secrets must be created before installation.
+#
+# This installation does not create an Ingress or LoadBalancer Service for
+# the Admin API. It requires port-forwards to access without further
+# configuration to add them, e.g.:
+# kubectl port-forward deploy/your-deployment-kong 8001:8001
+
+image:
+  repository: kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition
+  tag: 1.5.0.2-alpine
+  pullSecrets:
+    # you-must add this: https://github.com/Kong/charts/blob/master/charts/kong/README.md#kong-enterprise-docker-registry-access
+    - kong-enterprise-edition-docker
+
+admin:
+  enabled: true
+  http:
+    enabled: true
+    servicePort: 8001
+    containerPort: 8001
```

Comment:
1. Can we default to exposing Admin API on 8444 with TLS?
2. Does this configuration actually work? The controller will dial on 8444 by default and fail here, no?

Reply:
It works. This has used a template to choose the correct URL for a while now https://github.com/Kong/charts/blob/1.5.0/charts/kong/templates/_helpers.tpl#L121-L137

This example, by design, does very little beyond requiring the necessary secrets and exposing the admin API for use with Manager to economize on what you need to change. It's not intending to create a secure configuration; the full version is (though it's still limited in that respect because we don't support passing the certificates Kong serves as environment variables).

Using the self-signed certs is an option, but is likely to run into issues where requests from Manager fail unless you visit the admin API in your browser and click past its warning first.